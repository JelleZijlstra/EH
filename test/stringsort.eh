#!/usr/bin/ehi
# Test sorting strings
echo "foo" < "bar"
echo "foo" > "bar"

include '../lib/library.eh'
l = Cons "foo", Cons "bar", Cons "baz", Nil
echo l.sort()
