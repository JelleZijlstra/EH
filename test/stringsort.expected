false
true
bar::baz::foo::[]
